modules/fundamental/src/advisory/endpoints/test.rs

```diff
@@ -384,7 +384,11 @@ async fn upload_default_csaf_format(ctx: TrustifyContext) -> Result<(), anyhow::
             .service(web::scope("/api").configure(|svc| configure(svc, db, storage)));
     }))
     .await;
-    let payload = include_str!("../../../../../etc/test-data/csaf/cve-2023-33201.json");
+
+    let payload = include_str!(concat!(
+        env!("WORKSPACE_ROOT"),
+        "/etc/test-data/csaf/cve-2023-33201.json"
+    ));
```

Comment:
I'd prefer we use TrustifyContext, which is already capable of finding our test data:

```suggestion
    let payload = TrustifyContext::document_bytes("csaf/cve-2023-33201.json");
```

Reply:
I'm not sure we even need `document_{bytes,stream}` to be associated with `TrustifyContext`. We could bring them outside its impl (just like `find_workspace_root`) and import them directly when needed, e.g.
```
use trustify_test_context::document_bytes;
let payload = document_bytes("csaf/cve-2023-33201.json");
```

Reply:
this part is looking cool now, I added 2 basic tests just to make sure it works :+1: